frog/imports/ui/Analytics.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+ 
+export default class Analytics extends Component {
+  render() {
+    return (
+      <p>You are on the Analytics app</p>
+    );
+  }
+}
```

Comment:
For this I simply reproduced the way they do it in the Meteor + REACT tutorial. Furthermore These files are just the basis on which we will work and will become more complex in a near future.